recipe/meta.yaml

```diff
@@ -36,7 +36,9 @@ requirements:
     - setuptools_scm
     - setuptools_scm_git_archive
     - xarray
-    - python >=3.6
+    - rioxarray
+    - shapely <1.8.0
```

Comment:
shapely<1.8.0 it's a bug on holoview not in xsar and holoviz is not a xsar dependencies.
xsar run with all shapely version
I think it's a bug for holoviews user not for xsar user ?

Reply:
We are not able to run examples with `shapely>=1.8` . Examples need holoviews, and I was not able to install holoviews with shapely<1.8 once xsar was installed.
So pinning shapely<1.8 is a workaround for now. Hopefully, we will be able to unpin it once holoviews is fixed.

Reply:
It's developer dependancies or holoviews/xsar user, it's specific use case not for all users (matplotlib, xsar alone or other visualization lib)

In the documentation we can point out the problem for holoviews users. I think it's better to stay the most independent?

Reply:
Do you have another solution for xsar user to run examples ? Can you test it ?